de.dashup.application/src/main/webapp/views/includes/head.jsp

```diff
@@ -28,7 +28,22 @@
     <script type="text/javascript" src="../../libraries/classes/PostRequest.js"></script>
     <script type="text/javascript" src="../../libraries/index.js"></script>
 
-    <jsp:include page="webComponents.jsp" />
+    <%-- Web Components --%>
+    <script type="module">
+        import '../../components/dashup-grid-layout.js'
+        import '../../components/dashup-component.js';
+        import '../../components/dashup-text-input.js'
+        import '../../components/dashup-checkbox.js'
+        import '../../components/dashup-radio-button.js'
+        import '../../components/dashup-radio-button-group.js'
+        import '../../components/dashup-switch.js'
+        import '../../components/dashup-button.js'
+        import '../../components/dashup-display.js'
+        import '../../components/dashup-list.js'
+        import '../../widgets/dashup-share-banner.js'
+        import '../../widgets/dashup-todo.js'
+    </script>
```

Comment:
We might consider binding our components only in `index.jsp` and in the future also in the workbench. This improves the performance of all other pages.

Reply:
I agree.

Reply:
Fixed